scripts/zones/Carpenters_Landing/IDs.lua

```diff
@@ -44,6 +47,11 @@ zones[xi.zone.CARPENTERS_LANDING] =
         CRYPTONBERRY_EXECUTOR = GetFirstID('Cryptonberry_Executor'),
         MYCOPHILE             = GetFirstID('Mycophile'),
         HERCULES_BEETLE       = GetFirstID('Hercules_Beetle'),
+        PARA                  = GetFirstID('Para'),
+        PARA1                 = GetFirstID('Para') +1,
+        PARA2                 = GetFirstID('Para') +2,
+        PARA3                 = GetFirstID('Para') +3,
+        PARA4                 = GetFirstID('Para') +4,
```

Comment:
The can probably be `PARA` and `PARA_ADD_1`, `PARA_ADD_2` etc. Or you can just have `PARA` and then do your offsetting/lookups in the para script

Reply:
Updated.